app/models/cms/page.rb

```diff
@@ -91,25 +93,24 @@ def mime_type
   def to_xml(options = {})
     xml = options[:builder] || Nokogiri::XML::Builder.new
 
-    xml.page do |x|
+    xml.__send__(self.class.data_tag) do |x|
       unless new_record?
         x.id id
         x.created_at created_at.xmlschema
         x.updated_at updated_at.xmlschema
       end
-
       x.title title
+      x.section_id section_id
+      x.path path
```

Comment:
@jlledom Have you checked why it was `x.path(path)` before? Could it be that this pre-processing is still needed?

Reply:
Nevermind, sorry, I completely misread it before :facepalm: 

I somehow thought it was calling the `self.path` method https://github.com/3scale/porta/pull/3193/files#diff-9715ba077dc884135c880762dff0260b7b37c6ce7cd310855879c81551e4da35L33-L42

Of course it's not the case, but I am wondering what that method is for :thinking: It doesn't seem that it is used. But anyway, not part of this PR, we can clean up at some later point.

I am still wondering though, why in the original implementation it included `respond_to?(:path)` :thinking:  Any idea?

Reply:
No worries! About `respond_to?(:path)`, I found the commit: https://github.com/3scale/system/commit/2a41b095b5244b21fe3a6d7bd2f2c0f51f7efa0c. It doesn't explain why it calls `respond_to?` but it seems redundant to me, since it's a column name, it must exist.